Electron/src/__tests__/pages/PlaylistUser.test.tsx

```diff
@@ -522,3 +519,69 @@ test('Playlist user role update playlist', async () => {
 
   expect(refreshSidebarData).toHaveBeenCalledTimes(1);
 });
+
+test('Playlist updates song name in context when a song is clicked', async () => {
```

Comment:
Test is good, but we need to select the song card in another way (no testid) as said in the other comment